packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -252,21 +260,25 @@ export type ISummaryConfiguration =
 export const DefaultSummaryConfiguration: ISummaryConfiguration = {
     state: "enabled",
 
-    idleTime: 5000 * 3,
+    idleTime: 15 * 1000, // 15 secs.
 
-    maxTime: 5000 * 12,
+    maxTime: 60 * 1000, // 1 min.
 
-    maxOps: 1000, // Summarize if 1000 ops received since last snapshot.
+    maxOps: 100, // Summarize if 100 weighted ops received since last snapshot.
 
     minOpsForLastSummaryAttempt: 50,
 
-    maxAckWaitTime: 6 * 10 * 1000, // 6 min.
+    maxAckWaitTime: 6 * 10 * 1000, // 60 mins.
```

Comment:
I think we messed this up - it used to 600,000 
and it is now only 60 seconds (not 60 mins) 

Unless  Vlad asked to change it, it should be 600 * 1000  600 seconds or  10 mins